buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -2,12 +2,11 @@ object Versions {
     const val cosmos = "0.44.0"
     const val wasmd = "v0.19.0"
     const val pbProto = "1.7.0"
-    const val protobuf = "3.6.1"
-    const val protoUtil = "0.1.5"
+    const val protobuf = "3.19.2"
```

Comment:
I'm not sure if there's any negative implications of this.

Reply:
@scirner22 Yeah, I figured. I wanted to run it by you.

Reply:
I'm not familiar with the details, but I thought it was the case that doing this might make this unusable by most Figure services.

Reply:
I don't think bumping the protobuf version here would impact any services, but should be easy to verify. The other day I ran into an issue with using proto types using `oneof` and later versions of protobuf. I didn't dig too deep into fixing yet, but if the project still builds it should be fine

Reply:
sounds good then